Tests/web3swiftTests/remoteTests/EIP1559Tests.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  EIP1559Tests.swift
+//  
+//
+//  Created by Jann Driessen on 01.11.22.
+//
+
+import XCTest
+import Core
+
+@testable import web3swift
+
+final class EIP1559Tests: XCTestCase {
+
+    func testEIP1159MainnetTransaction() async throws {
+        let web3 = await Web3.InfuraMainnetWeb3(accessToken: Constants.infuraToken)
+        var tx = CodableTransaction(
+            type: .eip1559,
+            to: EthereumAddress("0xb47292B7bBedA4447564B8336E4eD1f93735e7C7")!,
+            chainID: web3.provider.network!.chainID,
+            value: Utilities.parseToBigUInt("0.1", units: .eth)!,
+            gasLimit: 21_000
+        )
+        // Vitalik's address
+        tx.from = EthereumAddress("0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B")!
+        // Should fail if there would be something wrong with the tx
+        let res = try await web3.eth.estimateGas(for: tx)
+        XCTAssertTrue(res > 0)
```

Comment:
Thank you, could you please change this assertion to XCAssertGreather then, those one gives much more clear output on fail.